wizeline/fox-poc-backend
# Feat/setup validation hooks

### What does this PR do?
Add validation of the content when creating a post.

### Where should the reviewer start?
At `src/services/posts/posts.hooks.js`

### How can this be manually tested?
Try to create a post without all the required fields (`author`, `title`, `content`, `status`, and `date` ) and check the response (should be invalid)
